cap_rds_schema.md

```diff
@@ -0,0 +1,28 @@
+# RDS file schema
+
+## AnnData fields mapping to Seurat5
+
+In the table below `seurat_obj` represents Seurat5 object, the result of the conversion. It consists of only one assay, which represents an `Assay5` object and is defined as `assay <- seurat_obs[["RNA"]]`
+
+| AnnData Component  | Seurat Component                       |
+|--------------------|----------------------------------------|
+| `adata.X`          | `assay@layers$data`                    |
+| `adata.var`        | `assay@meta.data`                      |
+| `adata.raw.X`*     | `assay@layers$counts`                  |
+| `adata.raw.var`*   | -                                      |
+| `adata.obs`        | `seurat_obj@meta.data`                 |
+| `adata.uns`        | `seurat_obj@misc`                      |
+| `adata.obsm`       | `seurat_obj@reductions`                |
+`*` - if exists
```

Comment:
`*` - if exists --> 

`*` - if it exists

Reply:
fixed